packages/vue-dot/src/patterns/RatingPicker/tests/RatingMixin.spec.ts

```diff
@@ -17,32 +13,57 @@ function createTestComponent() {
 	});
 }
 
-describe('RatingMixin', () => {
-	const label = 'Pourriez-vous donner une note ?';
+const labelQuestion = 'Pouvez-vous nous en dire plus ?';
+const itemLabels = ['non je dis rien','faut voir','je dirais tout ce que vous voulez'];
```

Comment:
```suggestion
const label = 'Pouvez-vous nous en dire plus ?';
const itemLabels = [
	'Non je dis rien',
	'Faut voir',
	'Je dirais tout ce que vous voulez'
];
```
Et est-ce que tu peux les déplacer dans le bloc `describe` stp ?